### PR TITLE
Fix issue with Dependabot ignore list

### PR DESCRIPTION
**Pull request by elenagarrone**
# Description

The current `"@storybook/*"` should work but it doesn't. I've found another person with the same issue but no answer.
I've updated the current list to be more specific which hopefully should work.

In addition to that I've modified the `.gitignore` to cover the nested node modules.

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts
- [ ] I have [linked the new component](<(https://github.com/Legal-and-General/canopy/blob/master/docs/CONTRIBUTING.md#invision-dsm)>) to adobe DSM (if appropriate)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,7 @@ storybook-static
 
 # dependencies
 /node_modules
```
Comment (thatguynamedandy) [inline .gitignore:15]:
```suggestion
node_modules/
```
Would this do the same thing?

**Review by thatguynamedandy — COMMENTED** [review]
One tiny but potential improvement.

File: .gitignore (hunk 1 above)
Comment (elenagarrone) [reply .gitignore]: Yes! updated thanks 👍

**Review by owensgit — APPROVED** [review]
Nice one @elenagarrone 👍

**Review by andy-polhill — APPROVED** [review]

**Pull request merged**